apps/site/assets/ts/stop/components/StationInformation.tsx

```diff
@@ -32,6 +36,9 @@ const StationInformation = ({
     }
   });
 
+  const elevators = facilitiesByType.get("ELEVATOR")!;
```

Comment:
**suggestion:** use the fallback `[]` here, e.g. change this to `facilitiesByType.get("ELEVATOR") || []`, which then simplifies passing this variable into `<ElevatorsAmenityCard>`. 

(same with escalators)